doc/motion_planning_api/src/motion_planning_api_tutorial.cpp

```diff
@@ -93,40 +100,44 @@ int main(int argc, char** argv)
   // We will get the name of planning plugin we want to load
   // from the ROS parameter server, and then load the planner
   // making sure to catch all exceptions.
-  if (!node_handle.getParam("planning_plugin", planner_plugin_name))
-    ROS_FATAL_STREAM("Could not find planner plugin name");
+  if (!motion_planning_api_tutorial_node->get_parameter("planning_plugin", planner_plugin_name))
+    RCLCPP_FATAL(LOGGER, "Could not find planner plugin name");
   try
   {
     planner_plugin_loader.reset(new pluginlib::ClassLoader<planning_interface::PlannerManager>(
         "moveit_core", "planning_interface::PlannerManager"));
   }
   catch (pluginlib::PluginlibException& ex)
   {
-    ROS_FATAL_STREAM("Exception while creating planning plugin loader " << ex.what());
+    RCLCPP_FATAL(LOGGER, "Exception while creating planning plugin loader %s", ex.what());
   }
   try
   {
     planner_instance.reset(planner_plugin_loader->createUnmanagedInstance(planner_plugin_name));
-    if (!planner_instance->initialize(robot_model, node_handle.getNamespace()))
-      ROS_FATAL_STREAM("Could not initialize planner instance");
-    ROS_INFO_STREAM("Using planning interface '" << planner_instance->getDescription() << "'");
+    if (!planner_instance->initialize(robot_model, motion_planning_api_tutorial_node,
+                                      motion_planning_api_tutorial_node->get_namespace()))
+      RCLCPP_FATAL(LOGGER, "Could not initialize planner instance");
+    RCLCPP_INFO(LOGGER, "Using planning interface '%s'", planner_instance->getDescription().c_str());
   }
   catch (pluginlib::PluginlibException& ex)
   {
     const std::vector<std::string>& classes = planner_plugin_loader->getDeclaredClasses();
     std::stringstream ss;
     for (const auto& cls : classes)
       ss << cls << " ";
-    ROS_ERROR_STREAM("Exception while loading planner '" << planner_plugin_name << "': " << ex.what() << std::endl
-                                                         << "Available plugins: " << ss.str());
+    RCLCPP_ERROR(LOGGER, "Exception while loading planner '%s': %s\nAvailable plugins: %s", planner_plugin_name.c_str(),
+                 ex.what(), ss.str().c_str());
   }
 
+  moveit::planning_interface::MoveGroupInterface move_group(motion_planning_api_tutorial_node, PLANNING_GROUP);
+
   // Visualization
   // ^^^^^^^^^^^^^
   // The package MoveItVisualTools provides many capabilities for visualizing objects, robots,
   // and trajectories in RViz as well as debugging tools such as step-by-step introspection of a script.
   namespace rvt = rviz_visual_tools;
-  moveit_visual_tools::MoveItVisualTools visual_tools("panda_link0");
+  moveit_visual_tools::MoveItVisualTools visual_tools(motion_planning_api_tutorial_node, "panda_link0",
+                                                      "motion_planning_api_tutorial", move_group.getRobotModel());
   visual_tools.loadRobotStatePub("/display_robot_state");
```

Comment:
Do you know if this line is still necessary?  It appeared in the previous version of the tutorial, so I left it in, but when I run the tutorial I get a warning that the topic `/display_robot_state` has no subscribers.

Reply:
It's probably an optional RViz plugin that can be turned on. I bet the rviz config just needs to be updated.

Reply:
I looked at the other tutorials that have been updated to use ROS 2, and none of them have this line.  They also got rid of the calls to `visual_tools.publishRobotState`, so I went ahead and removed them.

I also discovered that I could re-use `move_group.launch.py` (and the corresponding rviz config file) rather than creating `motion_planning_api_setup.launch.py`, which was almost an exact copy of `move_group.launch.py` anyway.